.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "A short description of the issue"
+labels: bug
+assignees: sinukaarel
```

Comment:
I would leave assignees out. If you'd assign the issue to yourself that would mean that you are working on it, which might not be the case.

Reply:
Removed https://github.com/sendsmaily/smaily-opencart-module/pull/79/commits/5baf85cd898f85b8f31042a4b0190670e752fdd2